src/lib/common/logTracing.cpp

```diff
@@ -55,12 +57,21 @@ void logInfoNotification
 void logInfoNotification
 (
   const char*  subId,
+  const char*  protocol,
+  const char*  endpoint,
   const char*  verb,
-  const char*  url,
+  const char*  resource,
   const char*  rc
 )
 {
-  LM_I(("Notif delivered (subId: %s): %s %s, response code: %s", subId, verb, url, rc));
+  if (strncmp(protocol, "mqtt:", strlen("mqtt:")) == 0)
+  {
+    LM_I(("MQTT Notif delivered (subId: %s): broker: %s, topic: %s", subId, endpoint, resource));
+  }
+  else
+  {
+    LM_I(("Notif delivered (subId: %s): %s %s%s, response code: %s", subId, verb, endpoint, resource, rc));
```

Comment:
```suggestion
    LM_I(("HTTP Notif delivered (subId: %s): %s %s%s, response code: %s", subId, verb, endpoint, resource, rc));
```

Make it sense to change the log to make it more "coherent" with HTTP notification?. Probably it would have an impact on tests (I am not sure)

Reply:
Ater talking offline, it could have some implications in backward compatibility, so NTC

Reply:
Yes, it could have impact in tests or existing operations procedures looking for trace patterns. Better not changing existing log.

Thus, NTC